### PR TITLE
docker: Add docker

### PR DESCRIPTION
**Pull request by pazembrz**
* INSPIR-2702

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+FROM inspirehep/poetry:1.0.0b1
+
+CMD [ "run", "inspire-disambiguation", "cluster" ]
+
+WORKDIR /opt/disambiguation
+
+COPY . .
+
+RUN poetry install
```
Comment (michamos) [inline Dockerfile:16]: so we're not doing the game of generating a `requirements.txt` and installing that with `pip` like for `inspirehep`?

File: inspire_disambiguation/config.py [hunk 2]
```diff
@@ -72,7 +72,6 @@
 
 instance_path = os.path.dirname(os.path.abspath(__file__))
 disambiguation_base_path = os.path.join(instance_path, "disambiguation")
-BASE_PATH = disambiguation_base_path
```
Comment (michamos) [inline inspire_disambiguation/config.py:75]: why? shouldn't the environment override this? or is the issue that all the other variables after have already been instanciated so you need to override them explicitly too?

Comment (pazembrz) [reply inspire_disambiguation/config.py:75]: Yes, this config file is parsed before env vars are loaded.

File: Dockerfile (hunk 1 above)
Comment (pazembrz) [reply Dockerfile:16]: We don't need it if we are installing all stuff from poetry like here.

File: inspire_disambiguation/config.py (hunk 2 above)
Comment (michamos) [reply inspire_disambiguation/config.py:75]: what I'm saying is that this is useless, because the other paths are hardcoded here too.

Comment (pazembrz) [reply inspire_disambiguation/config.py:75]: This is why I removed it?

File: .travis.yml [hunk 3]
```diff
@@ -1,20 +0,0 @@
...
-before_install:
-  - travis_retry pip install --upgrade pip setuptools py
-  - travis_retry curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python - --version 0.12.16;
-    source $HOME/.poetry/env;
-
-install:
-  - travis_retry poetry install
-
-script:
-  - "./run-tests.sh"
```
Comment (michamos) [inline .travis.yml:20]: so you remove travis, but you're not replacing it with drone here, are you? don't you want to do that simultaneously?

**Review by michamos — APPROVED** [review]

**Pull request merged**